sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/DeviceCapabilities.cs

```diff
@@ -16,12 +16,13 @@ public DeviceCapabilities()
         }
 
         /// <summary> Initializes a new instance of DeviceCapabilities. </summary>
-        /// <param name="iotEdge"> . </param>
+        /// <param name="iotEdge"> The property that determines if the device is an edge device or not. </param>
         internal DeviceCapabilities(bool? iotEdge)
         {
             IotEdge = iotEdge;
         }
 
+        /// <summary> The property that determines if the device is an edge device or not. </summary>
         public bool? IotEdge { get; set; }
```

Comment:
Probably should be `IsIotEdgeDevice`

Reply:
I'm not sure if changing this means a change for the service. They are only making changes that do not need any code change in their service.

Reply:
Fair. We could consider doing a customized name for the SDK. We'd need buy-in from Andrei, and then we'd want other lang SDKs to follow suit.

Reply:
This property can be overriden.